pkg/tenant/members.go

```diff
@@ -31,9 +35,40 @@ func (s *Server) MemberList(c *gin.Context) {
 	c.JSON(http.StatusNotImplemented, "not implemented yet")
 }
 
+// MemberDetail retrieves a summary detail of a member by its ID
+// and returns a 200 OK response.
+//
+// Route: /member/:memberID
 func (s *Server) MemberDetail(c *gin.Context) {
+	var (
+		err   error
+		reply *api.Member
+	)
 
-	c.JSON(http.StatusNotImplemented, "not implemented yet")
+	// Get the member ID from the URL and return a 400 if the member does not exist.
+	var memberID ulid.ULID
+	if memberID, err = ulid.Parse(c.Param("memberID")); err != nil {
+		log.Debug().Err(err).Msg("could not parse member ulid")
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("could not parse member id"))
+		return
+	}
+
+	// Get the specified member from the database and return a 404 response
+	// if it cannot be retrieved.
+	var member *db.Member
+	if member, err = db.RetrieveMember(c.Request.Context(), memberID); err != nil {
+		log.Error().Err(err).Msg("could not retrieve member")
```

Comment:
```suggestion
		log.Error().Err(err).Str("memberID", memberID.String()).Msg("could not retrieve member")
```

We might as well include this so we can tell which member ID the error is about.

Reply:
I added this to TenantDetail as well.